kirovy/migrations/0002_add_games.py

```diff
@@ -0,0 +1,153 @@
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from kirovy import constants
+from kirovy.models import CncGame
+
+
+def _forward(apps: StateApps, schema_editor: DatabaseSchemaEditor):
+    tib_dawn = CncGame.objects.create(
+        slug="td",
+        full_name=f"{constants.cnc_prefix} Tiberian Dawn",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=None,
+        is_mod=False,
+    )
+    tib_dawn.save()
+
+    red_alert = CncGame.objects.create(
+        slug="ra",
+        full_name=f"{constants.cnc_prefix} Red Alert",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=None,
+        is_mod=False,
+    )
+    red_alert.save()
+
+    dune_2k = CncGame.objects.create(
+        slug="d2",
+        full_name="Dune 2000",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=None,
+        is_mod=False,
+    )
+    dune_2k.save()
+
+    tib_sun = CncGame.objects.create(
+        slug="ts",
+        full_name=f"{constants.cnc_prefix} Tiberian Sun",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=None,
+        is_mod=False,
+    )
+    tib_sun.save()
+
+    red_alert_2 = CncGame.objects.create(
+        slug="ra2",
+        full_name=f"{constants.cnc_prefix} Red Alert 2",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=None,
+        is_mod=False,
+    )
+    red_alert_2.save()
+
+    yuri = CncGame.objects.create(
+        slug="yr",
+        full_name=f"{constants.cnc_prefix} Red Alert 2: Yuri's Revenge",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=red_alert_2,
+        is_mod=False,
+        compatible_with_parent_maps=True,
+    )
+    yuri.save()
+
+    dta = CncGame.objects.create(
+        slug="dta",
+        full_name="Dawn Of The Tiberium Age",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=tib_dawn,
+        is_mod=True,
+    )
+    dta.save()
+
+    mental_omega = CncGame.objects.create(
+        slug="mo",
+        full_name="Mental Omega",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=yuri,
+        is_mod=True,
+    )
+    mental_omega.save()
+
+    twisted_insurrection = CncGame.objects.create(
+        slug="ti",
+        full_name="Twisted Insurrection",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=tib_sun,
+        is_mod=True,
+    )
+    twisted_insurrection.save()
+
+    red_resurrection = CncGame.objects.create(
+        slug="yrrr",
+        full_name="YR Red-Resurrection",
+        is_visible=True,
+        allow_public_uploads=False,
+        parent_game=yuri,
+        is_mod=True,
+    )
+    red_resurrection.save()
+
+    cnc_reloaded = CncGame.objects.create(
+        slug="ccr",
```

Comment:
not sure if `ccr` or `cncr`, @FS-21?

Reply:
I'll do cncr. These will be changeable though, this is just initial seed data.

Reply:
fixed